poetry_private_repo_plugin/plugin.py

```diff
@@ -1,24 +1,46 @@
-from typing import List
+import fnmatch
+from typing import Dict, List, Optional
 
 from poetry.console.application import Application
 from poetry.core.packages.dependency import Dependency
 from poetry.core.packages.package import Package
+from poetry.core.pyproject.toml import PyProjectTOML
 from poetry.plugins.application_plugin import ApplicationPlugin
 from poetry.repositories.repository_pool import RepositoryPool
 
 
+class PluginConfig:
```

Comment:
For my future self: magic taken from here
https://github.com/artisanofcode/poetry-stickywheel-plugin/blob/main/src/poetry_stickywheel_plugin.py